Formula/r/rsyncy.rb

```diff
@@ -0,0 +1,40 @@
+class Rsyncy < Formula
+  include Language::Python::Virtualenv
+
+  desc "Status/progress bar for rsync"
+  homepage "https://github.com/laktak/rsyncy"
+  url "https://github.com/laktak/rsyncy/archive/refs/tags/v0.2.0.tar.gz"
+  sha256 "35b8998e5ff13edd5ea833fcb8b837013424d86c3110091028d3ca71cdd46bde"
+  license "MIT"
+  head "https://github.com/laktak/rsyncy.git", branch: "master"
+
+  depends_on "python@3.12"
+  depends_on "rsync"
+
+  uses_from_macos "zlib"
+
+  def python3
+    "python3.12"
+  end
```

Comment:
What is this for?

Reply:
sorry, leftover from switching from pyinstaller. I removed it.